### PR TITLE
Added the team structure and some debugging functionality for the client

### PR DESCRIPTION
**Pull request by djthorpe**
Added the methods "ListTeams" and "GetTeam" and also a simple test file. Also added a method for the client so you can switch debugging on.

### CODE REVIEW

File: clubhouse.go [hunk 1]
```diff
@@ -86,11 +100,24 @@ func (ch *Clubhouse) listResources(resource string) ([]byte, error) {
...
+		dump, err := httputil.DumpResponse(resp, true)
+		if err != nil {
+			return nil, err
+		}
+		fmt.Printf("%q\n", dump)
+	}
 	defer resp.Body.Close()
+	if ch.Debug {
+		fmt.Printf("<=%v\n", resp.Status)
+	}
```
Comment (jnormington) [inline clubhouse.go:120]: Could this not be part of the if ch.Debug if statement above ?

File: team_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import (
+	"io/ioutil"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	// Frameworks
+	ch "github.com/djthorpe/clubhouse-go"
```
Comment (jnormington) [inline team_test.go:11]: Import needs to updated

File: team_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+func getToken() (string, error) {
+	if user, err := user.Current(); err != nil {
+		return "", err
+	} else if bytes, err := ioutil.ReadFile(filepath.Join(user.HomeDir, ".clubhouse")); err != nil {
+		return "", err
+	} else {
+		return strings.TrimSpace(string(bytes)), nil
+	}
+}
```
Comment (jnormington) [inline team_test.go:48]: These I guess are more smoke tests :+1:

**Review by jnormington — CHANGES_REQUESTED** [review]
hey @djthorpe 

I'm so sorry I got no notification and was just browsing the repos. Just one change and I'll merge.
However seeing as its been almost 7months I'll understand if you don't want to make the changes, if not I'll update with your say so and merge.

**Comment by djthorpe** [comment]
Sorry I don'thave access to clubhouse anymore, so closing this pull request. Apologies.

**Pull request closed without merging**